web/services/getHolidaySummary.php

```diff
@@ -27,6 +27,7 @@
 
 $csvExport = array_key_exists("format", $_GET) && $_GET["format"] == "csv";
 $year = $_GET["year"] ?? NULL;
+$users = $_GET["users"] ?? NULL;
```

Comment:
I love how this could be combined with different kinds of front-end widgets to select a list of users from an area, a project, etc. I've been foreseeing something like this for ages, like in https://github.com/Igalia/phpreport/issues/445.